code/crates/app-channel/src/connector.rs

```diff
@@ -59,7 +59,9 @@ where
         match msg {
             HostMsg::ConsensusReady { reply_to } => {
                 let (reply, rx) = oneshot::channel();
-                self.sender.send(AppMsg::ConsensusReady { reply }).await?;
+                self.sender
+                    .send(crate::app_msg::ConsensusReady { reply }.into())
```

Comment:
All these `into`s here and below could be removed if we created a bespoke `struct AppChannelSender(tokio::mpsc::Sender)` with a method `send(msg: Into<AppMsg>`).

But that's outside the scope of this PR.